2_adventures/001/lenn4rd/03_raindrops_exercise/raindrops.rb

```diff
@@ -0,0 +1,25 @@
+class Fixnum
+  def prime_factor_of?(number)
+    number % self == 0
+  end
+end
+
+class Raindrops
+  PRIME_FACTORS = [3, 5, 7]
+
+  SOUNDS_FOR_FACTORS = {
+    '3' => 'Pling',
+    '5' => 'Plang',
+    '7' => 'Plong'
+  }
+  
+  def self.convert(number)
+    PRIME_FACTORS.each_with_object(sound = '') do |factor|
+      sound << SOUNDS_FOR_FACTORS[factor.to_s] if factor.prime_factor_of?(number)
+    end
+    
+    return number.to_s if sound.empty?
+    
+    sound
```

Comment:
What do you think about the third option?  (That I haven't written yet?)

` sound.emtpy? and number or sound`

Does that immediatey make more sense to as it reads compared to the ones with the non-word syntax?